From 29ae54839c57147936e659f421dc4fd57f162c8f Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Mon, 6 Jan 2020 09:02:37 +0000
Subject: [PATCH 244/245] net: ethernet: Replace TC_SETUP_MQPRIO with
 TC_SETUP_QDISC_MQPRIO

TC_SETUP_MQPRIO is old, now most drivers use TC_SETUP_QDISC_MQPRIO, so replace
it to align the latest use.

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/enetc.c   | 2 +-
 drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/enetc/enetc.c b/drivers/net/ethernet/freescale/enetc/enetc.c
index a19b6f36d8fa..b6a41a907497 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc.c
+++ b/drivers/net/ethernet/freescale/enetc/enetc.c
@@ -1499,7 +1499,7 @@ int enetc_setup_tc(struct net_device *ndev, enum tc_setup_type type,
 	u8 num_tc;
 	int i;
 
-	if (type != TC_SETUP_MQPRIO)
+	if (type != TC_SETUP_QDISC_MQPRIO)
 		return -EOPNOTSUPP;
 
 	mqprio->hw = TC_MQPRIO_HW_OFFLOAD_TCS;
diff --git a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
index 0c44e2b97a8d..08a21bc69843 100755
--- a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
+++ b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
@@ -1961,7 +1961,7 @@ static int dpaa2_eth_setup_tc(struct net_device *net_dev,
 	struct tc_mqprio_qopt *mqprio = (struct tc_mqprio_qopt *)type_data;
 	int i, err = 0;
 
-	if (type != TC_SETUP_MQPRIO)
+	if (type != TC_SETUP_QDISC_MQPRIO)
 		return -EINVAL;
 
 	if (mqprio->num_tc > dpaa2_eth_tc_count(priv)) {
-- 
2.17.1

